docs/autotuning.rst

```diff
@@ -192,6 +192,29 @@ flag is :code:`false`.
 
     options%transpose_use_inplace_buffers = .true.
 
+The :code:`autotune_transpose_skip` entry is an array of boolean flags that allows to skip certain
+transpose operations during autotuning. This option is meant for algorithms that only perform a subset
+of the four supported transposing operations. In this example, we autotune for the full set of
+operations, and therefore set all elements of :code:`autotune_transpose_skip` to :code:`false`.
+We should note that this is the default behavior, and thus there is no need to explicitly set
+the elements to :code:`false`.
+
+.. tabs::
+
+  .. code-tab:: c++
+
+    options.autotune_transpose_skip[0] = false; // do not skip X-to-Y transpose
```

Comment:
Thanks for updating the docs! Could you also add these lines to the `examples/cc/basic_usage/basic_usage_autotune.cu` and `examples/fortran/basic_usage/basic_usage_autotune.f90` to keep them consistent?

Reply:
I missed that! Updated in [a13338c](https://github.com/NVIDIA/cuDecomp/pull/16/commits/a13338ccd81867e6e591a28261fa77196bf4e8fb).